src/app/ReadHandler.cpp

```diff
@@ -46,6 +46,11 @@ ReadHandler::ReadHandler(ManagementCallback & apCallback, Messaging::ExchangeCon
     VerifyOrDie(apExchangeContext != nullptr);
 
     mExchangeCtx.Grab(apExchangeContext);
+#if CHIP_DEVICE_CONFIG_ENABLE_BOTH_COMMISSIONER_AND_COMMISSIONEE
```

Comment:
Ideally this would use a different define, which makes it clear that this is temporary and not a permanent feature of the "commissioner and commissioneee" configuration.

Reply:
the underlying issue is also cited in the comment

Reply:
Just to repeat: please make this a separate config value, one which makes it clear that the behavior being opted in to may have security implications.

Reply:
@chrisdecenzo And concretely, how about `CHIP_CONFIG_UNSAFE_SUBSCRIPTION_EXCHANGE_MANAGER_USE` for the define?